intel_realsense_grpc.cpp

```diff
@@ -354,6 +396,11 @@ const PipelineWithProperties startPipeline(const int colorWidth, const int color
     cout << "port:      " << selected_device.get_info(RS2_CAMERA_INFO_PHYSICAL_PORT) << "\n";
     cout << "usb type:  " << selected_device.get_info(RS2_CAMERA_INFO_USB_TYPE_DESCRIPTOR) << "\n";
 
+    float pix2mm_depth_scale = 1.0;
```

Comment:
```suggestion
    float depthScaleMm = 1.0;
```

let's try to use camel case in our code since we use that in most other places.

Reply:
I'd also use zero or NAN since it's not a valid value if depth is disabled.

Reply:
done